test/download-mongodb.ts

```diff
@@ -0,0 +1,129 @@
+/* This file has been adapted from
+ * https://github.com/mongodb-js/mongosh/blob/c8ddee2cc1f25db41774131895050492acdc7774/packages/build/src/download-mongodb.ts
+ * should be removed as part of https://jira.mongodb.org/browse/MONGOSH-1306
+ */
```

Comment:
It’s a bit unfortunate that this seems to be the easiest way to “just” get a MongoDB server binary downloaded but hopefully this will really go away with MONGOSH-1306.